README.md

```diff
@@ -123,6 +123,113 @@ export class UserRepository extends DefaultTransactionSoftCrudRepository<
 }
 ```
 
+The package also provides the following mixins which can be used for soft delete functionality:
+
+- **SoftDeleteEntityMixin**: This mixin adds the soft delete properties to your model. The properties added are represented by the IBaseEntity interface:
+
+```ts
+interface IBaseEntity {
+  deleted?: boolean;
+  deletedOn?: Date;
+  deletedBy?: string;
+}
+```
+
+There is also an option to provide config for the @property decorator for all these properties.
+Usage of SoftDeleteEntityMixin is as follows:
+
+```ts
+@model()
+class Item extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+    generated: true,
+  })
+  id?: number;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  name: string;
+
+  constructor(data?: Partial<Item>) {
+    super(data);
+  }
+}
+
+export class ItemSoftDelete extends SoftDeleteEntityMixin(Item, {
+  deletedBy: {name: 'deleted_by_userid'},
+}) {}
+```
+
+- **SoftCrudRepositoryMixin**: You can make use of this mixin to get the soft delete functionality for DefaultCrudRepository. You need to extend your DefaultCrudRepository with this mixin and provide DefaultCrudRepository class as input. You will have to inject the getter for IAuthUser in the contructor of your repository.
```

Comment:
User can extend any repository based on the DefaultCrudRepository na?

Reply:
Yes. Changed this. Also removed mixin for transactional repository as it extends default crud repository. SoftCrudRepositoryMixin is enough for both default and transaction crud repo.